packages/site/src/content/Disclosure/index.ts

```diff
@@ -21,4 +22,4 @@ export default {
       ),
     },
   ],
-};
+} as ContentExport;
```

Comment:
I think using [satisfies](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html) and changing this to `as const satisfies ContentExport` would be preferable. This would have the same outcome but would raise errors if for some reason what is defined here does not match ContentExport.